queries.sql

```diff
@@ -211,3 +211,10 @@ WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY COUNT(animals.id) DESC
 LIMIT 1;
+
+-- Week 2 DAY 1
+
+-- Take too much time
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits where animal_id = 4;
```

Comment:
- [ ] @Team Great work adding screenshot for your query however this query before execution time is below 1000ms, kindly consider adding more data before running it as shown below
![image](https://github.com/jkanyi-web/Vet-Clinic-DB/assets/61408860/255c0d48-d137-4578-a385-3fa5ca3d8de8)

![image](https://github.com/jkanyi-web/Vet-Clinic-DB/assets/61408860/e51337cf-945d-462b-85db-8b264d2cc81d)

Reply:
- [ ] @Team I discovered your after screenshot doesn't show the execution time, kindly consider retaking it to ensure it shows the difference in execution before and after improving te execution time.
![image](https://github.com/jkanyi-web/Vet-Clinic-DB/assets/61408860/8e75ea00-72a1-445d-a848-affc38a94aeb)